miso-web/src/main/webapp/scripts/freezer_ajax.js

```diff
@@ -48,6 +48,9 @@
   };
   
   Freezer.addRoom = function () {
+    Freezer.addRoomWithCallback(function() { Utils.showOkDialog('Room created', []); });
+  },
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove use of this comma operator. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS878)